### DISCUSSION TITLE
What 8.x packages are supported for .NET 6?

### DISCUSSION THREAD

**Discussion by pergardebrink** [dotnet/runtime#96977] [Q&A]
Since many packages, and specifically the Microsoft.Extensions ones, are following the .NET version when a new version is released, are they all supposed to be supported fully on previous versions as well (and in some cases even netstandard2.0/.NET Framework 4.6.2)?

Is there a matrix showing any incompatibilities, if any, or can I start using the 7.x and 8.x versions from .NET 6 before I upgrade to .NET 8 as long as their targets are for my version?

**Related question:** 
Is there a list of those packages that _are_ following the .NET version? 
There are some packages which seems to do that, for example to name a few, based on what I saw based on some dependencies we have: 

- System.Collections.Immutable
- System.ServiceModel.Syndication
- System.Runtime.Caching
- System.Security.Cryptography.Xml
- System.Text.Encodings.Web

**Comment by neon-sunset** [comment]
The list of supported versions is indicated by supported TFMs. For example, `System.Collections.Immutable` will work on as low as .NET Framework 4.6.2: https://www.nuget.org/packages/System.Collections.Immutable#supportedframeworks-body-tab

It would be strange if a particular target was supported by a package only for it to not work there :)
